taskiq/cli/scheduler/args.py

```diff
@@ -32,7 +34,21 @@ def from_cli(cls, args: Optional[Sequence[str]] = None) -> "SchedulerArgs":
             formatter_class=ArgumentDefaultsHelpFormatter,
             description="Subcommand to run scheduler",
         )
-        parser.add_argument("scheduler", help="Path to scheduler")
+        parser.add_argument(
```

Comment:
We don't need two parameters. Specify only factory. We can check if the imported scheduler is a factory after we import it.